test/integration/organisation_test.rb

```diff
@@ -6,523 +6,13 @@ class OrganisationTest < ActionDispatch::IntegrationTest
   before do
     org_example = GovukSchemas::Example.find("organisation", example_name: "organisation")
 
-    @content_item_no10 = {
-      title: "Prime Minister's Office, 10 Downing Street",
-      description: "10 Downing Street is the official residence of the British Prime Minister.",
-      base_path: "/government/organisations/prime-ministers-office-10-downing-street",
-      details: {
-        body: "10 Downing Street is the official residence and the office of the British Prime Minister.",
```

Comment:
This line is why the updated tests failed: the body is actually rendered html, so this example should have been wrapped in `<div class="govspeak"><p>` tags. The change to use the schemas is 👍

Reply:
I noted you wrote a nice commit message for this - I was just noting as I went along